src/controller/login.js

```diff
@@ -16,7 +16,7 @@ const refreshToken=async(req,res)=>{
       const newToken=await authservice.refreshToken(email,password)
```

Comment:
camelCase needed and newToken: newToken is not required. Just newToken is enough.